### PR TITLE
Updating the dev-spaces command from aks group to ads group

### PR DESCRIPTION
**Pull request by saurabsa**
- refactoring

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `./scripts/ci/test_static.sh` locally? (`pip install pylint flake8` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli-extensions/blob/master/docs/extension_summary_guidelines.md).

### CODE REVIEW

File: src/dev-spaces-preview/azext_dev_spaces_preview/__init__.py [hunk 1]
```diff
@@ -19,19 +20,19 @@ def __init__(self, cli_ctx=None):
                                                         min_profile='2017-03-10-profile')
 
     def load_command_table(self, _):
-        with self.command_group('aks') as g:
-            g.custom_command('use-dev-spaces', 'aks_use_dev_spaces')
-            g.custom_command('remove-dev-spaces', 'aks_remove_dev_spaces')
+        with self.command_group('ads') as g:
```
Comment (stepro) [inline src/dev-spaces-preview/azext_dev_spaces_preview/__init__.py]: I thought the plan was to make these internal (so they don't appear as actual commands in az).

Comment (saurabsa) [reply src/dev-spaces-preview/azext_dev_spaces_preview/__init__.py]: That's the next update when we have other commands. Right not the extension should have some exposed commands to work with.

File: src/dev-spaces-preview/azext_dev_spaces_preview/azext_metadata.json [hunk 2]
```diff
@@ -1,4 +1,4 @@
 {
-    "azext.minCliCoreVersion": "2.0.27",
+    "azext.minCliCoreVersion": "2.0.32",
```
Comment (williexu) [inline src/dev-spaces-preview/azext_dev_spaces_preview/azext_metadata.json]: This PR relies on the CLI's next release right?
If so, change this to `2.0.33.dev0`

**Comment by williexu** [comment]
Merging this, open a new PR for version bump/index.json update

**Pull request merged**